src/PIL/features.py

```diff
@@ -51,7 +51,8 @@ def get_supported_codecs():
     "webp_anim": ("PIL._webp", 'HAVE_WEBPANIM'),
     "webp_mux": ("PIL._webp", 'HAVE_WEBPMUX'),
     "transp_webp": ("PIL._webp", "HAVE_TRANSPARENCY"),
-    "raqm": ("PIL._imagingft", "HAVE_RAQM")
+    "raqm": ("PIL._imagingft", "HAVE_RAQM"),
+    "libjpegturbo": ("PIL._imaging", "HAVE_LIBJPEGTURBO")
```

Comment:
Nit: append a comma so the next addition is only a one-line diff: 

```suggestion
    "libjpegturbo": ("PIL._imaging", "HAVE_LIBJPEGTURBO"),
```